app/lib/feed_manager.rb

```diff
@@ -354,6 +355,13 @@ def blocks_or_mutes?(receiver_id, account_ids, context)
   def filter_from_home?(status, receiver_id, crutches)
     return false if receiver_id == status.account_id
     return true  if status.reply? && (status.in_reply_to_id.nil? || status.in_reply_to_account_id.nil?)
+    # hometown: exclusive list rules
+    unless timeline_type == :list
+      # find all exclusive lists
+      @list = List.where(account: Account.find(receiver_id), is_exclusive: true)
+      # is there a list the receiver owns with this account on it? if so, return true
+      return true if ListAccount.where(list: @list, account_id: status.account_id).exists?
```

Comment:
```suggestion
      lists = List.where(account_id: receiver_id, is_exclusive: true)
      # is there a list the receiver owns with this account on it? if so, return true
      return true if ListAccount.where(list: lists, account_id: status.account_id).exists?
```
As per https://github.com/hometown-fork/hometown/commit/bfc0fbab2c94dc134715f555edf793991da172a2#r54394746 and https://github.com/glitch-soc/mastodon/pull/1293/commits/b46dc2da243119414cb14ba3b74060f441da81b9